documentation/src/main/asciidoc/manual-index.asciidoc

```diff
@@ -895,30 +898,3 @@ operational. Here’re some ideas:
   the end.
 - The new mass indexer is not able to handle entity types with composite PK.
 - JPQL
-
-[[preface]]
-
-[preface]
-== Preface
-
-This is a Google Summer of Code 2016 subject.
-
-Mentors: Gunnar Morling and Emmanuel Bernard | Student: Mincong Huang
```

Comment:
A shame to lose these credits.
Could you (in a separate commit) add a your name at the top of index.asciidoc, just after "Sanne Grinovero"?
Also, you can add it to copyright.txt at the root of the repository, just after "Michael J. Simons". We'll do it ourselves eventually, but what's done no longer needs doing :)